piqueserver/commands.py

```diff
@@ -347,6 +351,25 @@ def get_truthy(value):
 
 
 def handle_command(connection, command, parameters):
+    """
+    Public facing function to run a command, given the connection, a command
+    name, and a list of parameters.
+
+    Will log the command.
+    """
+    result = _handle_command(connection, command, parameters)
+
+    if result == False:
+        parameters = ['***'] * len(parameters)
```

Comment:
This might be a good reason to introspect why this being done at all?

Also note that we need to be careful here with logging secret info such as passwords. Perhaps commands could receive a flag they can use to censor the parameters?

Reply:
Agreed. I did have a poke around and couldn't find any core commands that returned `False`. This behaviour should at least be documented.

Reply:
Oh, so returning False is what censors the commands... I'm not convinced that's the way we should go. I think I'd personally add a decorator that sets some flag on the command function.